docs/paper_JOSS/paper.md

```diff
@@ -0,0 +1,106 @@
+---
+title: TorchMetrics - Measuring Reproducibility in PyTorch
+tags:
+  - python
+  - deep learning
+  - pytorch
+authors:
+  - name: Nicki Skafte Detlefsen
+    affiliation: '1,2'
+    orcid: 0000-0002-8133-682X
+  - name: Jiri Borovec
+    affiliation: '1'
+    orcid: 0000-0001-7437-824X
+  - name: Justus Schock
+    affiliation: '1,3'
+    orcid: 0000-0003-0512-3053
+  - name: Ananya Harsh Jha
+    affiliation: '1'
+  - name: Teddy Koker
+    affiliation: '1'
+  - name: Luca Di Liello
+    affiliation: '4'
+  - name: Daniel Stancl
+    affiliation: '5'
+  - name: Changsheng Quan
+    affiliation: '6'
+  - name: William Falcon
+    affiliation: '1,7'
+affiliations:
+  - name: Grid AI Labs
+    index: 1
+  - name: Technical University of Denmark
+    index: 2
+  - name: University Hospital Düsseldorf
+    index: 3
+  - name: University of Trento
+    index: 4
+  - name: Charles University
+    index: 5
+  - name: Zhejiang University
+    index: 6
+  - name: New York University
+    index: 7
+date: 08 Dec 2021
+bibliography: paper.bib
+---
+
+# Summary
+
+Among recent machine learning publications, many cannot be reproduced [@reproducibility]. This is often due to different metrics implementations, which may lead to important training mechanisms like checkpointing, learning rate schedules, or early stopping to behave differently. For example, complex metrics like Fréchet inception distance (FID) for synthetic image quality evaluation [@fid] may differ widely based on the specific interpolation method used.
+
+A metric[^1] hereby is an objective quantitative measure of the model’s performance. To tackle the problem of reproducibility, there are initiatives such as Papers With Code [@papers_with_code], which link the research code with the corresponding papers. However, not all authors provide their code openly accessible. Therefore, we aim to tackle the problem another way - We believe that unified implementations of the relevant metrics can help overcome reproducibility by providing a de-facto reference implementation.
```

Comment:
hereby is unnecessary (don't add unnecessary language)

Reply:
not all authors provide their code openly accessible -> not all authors make opensource their code

Reply:
remove (herefore, we aim to tackle the problem another way -) (unecessary)..